### ISSUE TITLE
UseSpacingSniff doesn't catch superfluous spacing between same types

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#324]
```php
<?php

declare(strict_types=1);

namespace X;

use Foo;
use Bar;

use Baz;

use function blah;
```

The sniff will only report and fix:
```php
Expected 0 lines between different types of use statement, found 1.
```
No error for superfluous line between Bar and Baz.

Not quite sure if this is bug report or feature request, either way it would be nice behavior imho. :)

**Comment by kukulich** [comment]
Implemented in https://github.com/slevomat/coding-standard/commit/ac5adeae3f77f0a5971ada5e6fa48f136d3ea0c8 :)
